src/form/middleware.ts

```diff
@@ -0,0 +1,155 @@
+import { create } from '@dojo/framework/core/vdom';
```

Comment:
Looking at [Tom's PR](#803) it seems he tested the middleware as well. Might be something to consider here

Reply:
Yeah I can do that. It will be almost an exact copy of the widget version (which does fully cover the middleware) but probably a good idea regardless.